compiler-rt/lib/radsan/tests/radsan_test_context.cpp

```diff
@@ -68,14 +68,14 @@ TEST(TestRadsanContext,
 }
 
 TEST(TestRadsanContext, onlyDiesIfExitWithFailureReturnedFromUser) {
-  auto fake_action = radsan::OnErrorAction::Continue;
-  auto action_getter = [&fake_action]() { return fake_action; };
 
-  auto context = radsan::Context{action_getter};
+  setenv("RADSAN_ERROR_MODE", "continue", 1);
```

Comment:
Instead of injecting the action_getter, we skip the middle man and change the environment variables.

This makes this test a little less unit-test, and a little more integration test, for better or for worse.